packages/builder/src/types.ts

```diff
@@ -83,6 +83,8 @@ export interface ChainBuilderContext extends PreChainBuilderContext {
   extras: { [label: string]: string };
 
   imports: BundledChainBuilderOutputs;
+
+  [additionalProperties: string]: any;
```

Comment:
call this perhaps `shortContract`? and you may be able to define the type more precisely here than `any` for the value!